pkg/api/pod/util_test.go

```diff
@@ -170,6 +170,10 @@ func collectResourcePaths(t *testing.T, resourcename string, path *field.Path, n
 	case reflect.Ptr:
 		resourcePaths.Insert(collectResourcePaths(t, resourcename, path, name, tp.Elem()).List()...)
 	case reflect.Struct:
+		// Specifically skip ObjectMeta because it has recursive types
```

Comment:
Nit: include the name of the recursive field (ManagedFields) - though this will be more obvious once the PR has merged :-)